src/main/python/systemml/mllearn/estimators.py

```diff
@@ -29,6 +29,8 @@
 from sklearn.metrics import accuracy_score, r2_score
 from py4j.protocol import Py4JError
 import traceback
+import keras
+import keras2caffe
```

Comment:
No need for the `keras` import here since we're not using keras code directly in this file.  As for the keras2caffe import, `import keras2caffe` would try to import an installed package called keras2caffe.  Instead, we need to use a relative import, i.e., `from . import keras2caffe`.  Also, since this currently relies on Caffe, let's lazily import `keras2caffe` for now at the top of `__init__` constructor of the `Keras2DML` class, with a note up here that indicates we are doing that until the Caffe dependency is removed.  That way, an existing user can continue to import `mllearn` without needing Caffe.